docs/pages/docs/styling-theming/overriding-css.mdx

```diff
@@ -9,6 +9,8 @@ import { Example } from "@/components/example";
 
 # Overriding CSS
 
+TODO: explain or link to which classes are exposed
```

Comment:
I think if we want to add this, we should really limit which classes are mentioned since there are a lot of `bn-xyz` classes. Probably just want to mention some of the following:

`bn-container`: Element for whole editor
`bn-editor`: Element for the editor only, no toolbars or menus
`bn-block`: Element for block + nested blocks
`bn-group`: Element for nested blocks
`bn-inline-content`: Element for block content
`bn-toolbar`: Element for formatting & hyperlink toolbars
`bn-side-menu`: Element for side menu
`bn-drag-handle-menu`: Element for drag handle menu
`bn-slash-menu`: Element for slash menu

+ Say that Mantine selectors are also supported